lib/v2/screens/sign_up/viewmodels/signup_bloc.dart

```diff
@@ -14,10 +15,13 @@ part 'signup_state.dart';
 class SignupBloc extends Bloc<SignupEvent, SignupState> {
   SignupBloc({
     required ClaimInviteUseCase claimInviteUseCase,
+    required CreateUsernameUseCase createUsernameUseCase,
```

Comment:
We are not using DY in our blocs.

Reply:
I have been using this pattern in my prior PRs as well and I think it's a better practice to use inject these dependencies. If you guys strongly believe it should be done differently we can address this and other places in a separate PR.

Reply:
> I have been using this pattern in my prior PRs as well and I think it's a better practice to use inject these dependencies. If you guys strongly believe it should be done differently we can address this and other places in a separate PR.
Sounds good address a different PR to this what do you think @gguijarro-c-chwy ?

Reply:
I like DI, I will leave this up to the developer discretion for now, until we make a call as a team.